beacon-chain/blockchain/process_block_test.go

```diff
@@ -737,89 +736,6 @@ func TestEnsureRootNotZeroHashes(t *testing.T) {
 	assert.Equal(t, root, r, "Did not get wanted justified root")
 }
 
-func TestFinalizedImpliesNewJustified(t *testing.T) {
```

Comment:
Is this logic still something that needs to be tested post-refactor? If so, where is it covered?

Reply:
nope, `finalizedImpliesNewJustified` was fully deprecated and removed